uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -0,0 +1,74 @@
+@GGAD
+Feature: GGMQ-1
+
+  As a developer, I can configure my GGAD devices to connect to the local GGC using cloud based discovery
+
+  Background:
+    Given my device is registered as a Thing
+    And my device is running Greengrass
+    And the control component configured:
+    """
+    {
+        "port": 47619
+    }
+    """
+
+  Scenario: GGMQ-1-T1: As a customer, I can associate and connect a single GGAD with a single GGC based on cda configuration
+    When I create a Greengrass deployment with components
+      | aws.greengrass.clientdevices.Auth        | LATEST                                                        |
+      | aws.greengrass.clientdevices.mqtt.EMQX   | LATEST                                                        |
+      | aws.greengrass.client.Mqtt5JavaSdkClient | classpath:/greengrass/components/recipes/client_java_sdk.yaml |
+    Then the Greengrass deployment is COMPLETED on the device after 300 seconds
+    When I add "agent1" ggad with the following policy to the CDA configuration
```

Comment:
I suggest  misuse agentID as thingName here. ThingName (equals to clientId in term of MQTT proto) specified when control request client to make MQTT connection.

Reply:
Let's discuss naming conventions